optuna/integration/sklearn.py

```diff
@@ -4,47 +4,38 @@
 from numbers import Integral
 from numbers import Number
 from time import time
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Iterable
+from typing import List
+from typing import Mapping
+from typing import Optional
+from typing import Union
 
 import numpy as np
+import pandas as pd
```

Comment:
``pandas`` is not listed in the requirements of ``optuna``. Please remove this line and add the following block before the import block of ``sklearn``.

```python

with try_import():
    import pandas as pd
```

Reply:
Thank you for reviewing. Applied!